client/systems.go

```diff
@@ -163,3 +163,46 @@ func (client *Client) SystemDetails(seedLabel string) (*SystemDetails, error) {
 	}
 	return &rsp, nil
 }
+
+type InstallStep string
+
+const (
+	InstallStepSetupStorageEncryption InstallStep = "setup-storage-encryption"
+	InstallStepFinish                 InstallStep = "finish"
```

Comment:
What does the "finish" step do? A description on why this step is needed might help.

Reply:
Thanks, happy to add more details. I will do that in a followup if that's okay so that this foundational work can land. I have something like this in mind (closely following our sd134 spec):
```
// Creates a change to finish the installation. The change ensures all volume structure content is written to disk, it sets up boot, kernel etc and when finished the installed system is ready for reboot.
```

Reply:
Similar to setup-storage-encryption

Reply:
I did this now in https://github.com/snapcore/snapd/pull/12128

Reply:
Thank you!